canonical-web-and-design/canonical.com
# Merge search branch to the main

## Done

Add search on the careers pages

## QA

- Go to https://canonical-com-587.demos.haus/careers/all and also `/careeers/<department>`
- Check the auto complete works well 
- Check the right job list displays after pressing the enter
-  Check the filters work with the search
